build.sbt

```diff
@@ -20,22 +20,33 @@ scalacOptions ++= Seq(
   "-Ywarn-numeric-widen" // Warn when numerics are widened.
 )
 
+// Added to get info about Java warnings (compiling with JDK 8u47) AKa270116
+//
+javacOptions ++= Seq(
+  "-Xlint:unchecked"
+)
+
```

Comment:
Added those to get more info on certain Java warnings (the compiler suggested doing this). Didn't try cleaning up the warnings.